Basket-Donation-Payments/Basket-Donation-Payments/View Controllers/BasketViewController.m

```diff
@@ -62,10 +64,21 @@ - (void)favoriteUnfavoriteBasket {
         self.basket.favoriteCount += 1;
     }
     self.numberOfFavesLabel.text = [NSString stringWithFormat:@"%ld Favorites", (long)self.basket.favoriteCount];
+
+    [self updateBasketFeaturedValueWeights];
     [self.basket saveInBackground];
     [[User currentUser] saveInBackground];
 }
 
+- (void)updateBasketFeaturedValueWeights {
+    FeaturedValueWeight *ftWeights = self.basket.featuredValueWeights;
+    NSUInteger maxUserFavoritesWeight = 30;
+    ftWeights.userFavoritesWeight = MIN(self.basket.favoriteCount, maxUserFavoritesWeight);
```

Comment:
Same here, a basket with 100K likes has the same weight as a basket with 30 likes, is that correct?